Sources/SwiftWin32/Platform/Win32+Menu.swift

```diff
@@ -12,8 +12,8 @@ internal final class _MenuBuilder: MenuSystem {
 
   private var menus: OrderedSet<Menu>
 
-  internal init?(for view: View) {
-    self.hMenu = MenuHandle(owning: CreateMenu())
+  internal init?(for view: View, asWindowMenu: Bool = false) {
```

Comment:
Do we really need the `asWindowMenu` option?  Can you do a context menu on `Window`?  If not, we _could_ get away with `view is WIndow ? CreateMenu() : CreatePopupMenu()`.  I think that tweaking this way might be nicer:

```suggestion
  internal init?(for view: View, popup: Bool = false) {
```

Reply:
Yeap, I think you're right, we don't really need this argument.